submission/distributor_test.go

```diff
@@ -102,10 +118,75 @@ func TestNewDistributorLogClients(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			_, err := NewDistributor(tc.ll, ctpolicy.ChromeCTPolicy{}, tc.lcBuilder)
 			if gotErr, wantErr := err != nil, tc.errRegexp != nil; gotErr != wantErr {
-				t.Errorf("Expected error state does not match produced one.")
+				t.Errorf("Got error = %v, expected error = %v", (err == nil), (tc.errRegexp == nil))
```

Comment:
```suggestion
				t.Errorf("Got error = %v, expected error = %v", gotErr, wantErr)
```
As it was, this would print out "Got error = true" when err == nil. Also, you might find the error message more helpful if it contained the error message (when err != nil).

Reply:
Updated. Thanks for the catch